src/components/Filters.vue

```diff
@@ -127,12 +127,6 @@ export default defineComponent({
         mdIcon: albumsOutline,
         enumTypeId: "INVENTORY_SYS_JOB"
       },
-      {
```

Comment:
I think we should not remove this filters, as we might have jobs with this enum type and we need to filter those.